README.md

```diff
@@ -1413,4 +1413,8 @@ I'm [Yury](https://github.com/yuryzaytsev). This is a street cat named Bob. He i
 My name is [Dima](https://github.com/youhymuk) and I'm glad to study here with you.
 I wish you all success!
 
-![kottan](gif/cat-headshot.gif)
\ No newline at end of file
+![kottan](gif/cat-headshot.gif)
+
+### Greetings kottans
```

Comment:
Hi! Good job!
But there was no need to close previous PRs

Reply:
I was try to delete 1416 line with different cases :-) Sorry for deleting previous PRs. This won't help) Now I know :-)